_posts/2023-03-15-trace-analytics-jaeger.md

```diff
@@ -0,0 +1,41 @@
+
+## Jaeger
+
+One of the most popular open source solutions for distributed tracing is [Jaeger](https://www.jaegertracing.io/). Jaeger is an open source, end-to-end solution, hosted by the Cloud Native Computing Foundation (CNCF). Jaeger SDKs are OpenTelemetry (OTel) based and support multiple open source data stores, such as Cassandra and OpenSearch, for storing traces. OpenSearch now provides the option for visualizing traces in the OpenSearch Dashboards, which is the native visualization tool included in OpenSearch.
+
+## Trace analytics
+
+OpenSearch provides extensive support for log analytics and observability use cases. Starting with version 1.3, OpenSearch has added support for analyzing distributed tracing data with its Observability feature. Using Observability, you can analyze the crucial Rate, Error, Duration (RED) metrics in their trace data. Additionally, you can evaluate various components of your system for latency and errors and pinpoint services that need attention.
+
+OpenSearch trace analytics launched with support for OTel-compliant trace data provided by Data Prepper---the OpenSearch server-side data collector. To incorporate the popular Jaeger trace data format, in version 2.5, OpenSearch introduced the Trace analytics feature in OpenSearch Observability. 
+
+With OpenSearch Observability, you can now filter traces to isolate the spans that are showing errors in order to quickly identify the relevant logs. You can use the same feature-rich analysis capabilities for RED metrics, contextually linking traces and spans to their related logs that are available for the Data Prepper trace data.
+
+Keep in mind that OTel and Jaeger formats have several differences, as outlined in [OpenTelemetry to Jaeger Transformation](https://opentelemetry.io/docs/reference/specification/trace/sdk_exporters/jaeger/).
+
+## Try it out
+
+To try out this new feature, see the [Analyzing Jaeger trace data](https://opensearch.org/docs/latest/observing-your-data/trace/trace-analytics-jaeger/) documentation. The documentation includes a docker compose file that shows you how to add sample data using the Jaeger hot rod demo and then visualize it using trace analytics. To enable this feature, you need to set the `--es.tags-as-fields.all` flag to `true`, as described in the related [GitHub issue](https://github.com/jaegertracing/jaeger/issues/1299). This is necessary because of an [OpenSearch Dashboards limitation](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/657).
+
+In Dashboards, you can see the top service and operation combinations with the highest number of errors and latency. Selecting any of these will automatically bring you to the **Traces** page with the appropriate filters applied. You can also investigate any trace or service on your own by applying various filters.
```

Comment:
Confirm that my first change retains technical accuracy. We also need an "..., as shown in the following image" in the appropriate place.